save-frontend-common/src/main/kotlin/com/saveourtool/save/frontend/common/components/basic/UserBoard.kt

```diff
@@ -0,0 +1,69 @@
+/**
+ * Components for cards
+ */
+
+@file:Suppress("FILE_NAME_MATCH_CLASS", "MatchingDeclarationName")
+
+package com.saveourtool.save.frontend.common.components.basic
+
+import com.saveourtool.save.info.UserInfo
+import com.saveourtool.save.validation.FrontendRoutes
+
+import react.FC
+import react.Props
+import react.dom.html.ReactHTML.div
+import react.dom.html.ReactHTML.figure
+import web.cssom.ClassName
+import web.cssom.Length
+import web.cssom.rem
+
+/**
+ * A functional component to display users' avatars.
+ */
+@Suppress("MAGIC_NUMBER")
+val userBoard: FC<UserBoardProps> = FC { props ->
+    div {
+        className = ClassName("latest-photos")
+        div {
+            className = ClassName("row")
+            props.users.forEach { user ->
+                div {
+                    className = ClassName(props.avatarOuterClasses.orEmpty())
+                    figure {
+                        renderAvatar(user, props.avatarInnerClasses.orEmpty(), "/${FrontendRoutes.VULN_PROFILE}/${user.name}") {
```

Comment:
I am afraid it will be difficult for him now, this PR is still not finished - my suggestion to do that, but in a separate PR asap as soon as he will merge this one and we can unlock other people who are blocked by Vlad now